pills/01-why-you-should-give-it-try.xml

```diff
@@ -10,50 +10,50 @@
     <title>Introduction</title>
 
     <para>
-        Welcome to the first post of the <link
-        xlink:href="https://nixos.org/nix">Nix</link> in pills series.
-        Nix is a purely functional package manager and deployment
-        system for POSIX. There's a lot of documentation that
-        describes what Nix, <link
-        xlink:href="https://nixos.org/nixos">NixOS</link> and related
-        projects are.
+      Welcome to the first post of the "<link
+      xlink:href="https://nixos.org/nix">Nix</link> in pills" series.
+      Nix is a purely functional package manager and deployment
+      system for POSIX.
     </para>
 
     <para>
-      The purpose of this post is to convince you to give Nix a try.
+      There's a lot of documentation that describes what Nix, <link
+      xlink:href="https://nixos.org/nixos">NixOS</link> and related
+      projects are.
+      But the purpose of this post is to convince you to give Nix a try.
```

Comment:
Did you want this to be a new para, or just trying to not disrupt the formatting of the rest of the lines?

Reply:
The latter.